src/shared/conf-parser.c

```diff
@@ -45,6 +45,8 @@
 #include "syslog-util.h"
 #include "time-util.h"
 #include "utf8.h"
+#include "dns-type.h"
+#include "resolved-manager.h"
```

Comment:
as per coding style, please sort includes alphabetically

Reply:
also, please do *not* include resolved-specific code in our generic (shared/) code.